### DISCUSSION TITLE
Disable Zooming via CTRL + Mouse Wheel

### DISCUSSION THREAD

**Discussion by dbuechel** [cefsharp/CefSharp#4732] [General]
We have in Safe Exam Browser configuration options to control whether a user is allowed to zoom a page or not. This works very well so far (using the [SetZoomLevel(..)](https://cefsharp.github.io/api/118.6.x/html/M_CefSharp_WebBrowserExtensions_SetZoomLevel_1.htm) extension method and controls in our own user interface).

However, we have received reports that users are still able to zoom even when they're not supposed to do so, using either the touch pad of a mobile device or `CTRL + Mouse Wheel`. The pinch zooming via touch pad seems to be controllable with the following command-line argument for CEF / Chromium:

```
cefSettings.CefCommandLineArgs.Add("disable-pinch");
```

Now our question is whether there is anything similar to control the ability to zoom via `CTRL + Mouse Wheel`?

**Comment by amaitland** [comment]
If you are using `WinForms` then I'm not aware of an easy way to achieve this.

Search/ask on https://www.magpcss.org/ceforum/index.php to see if anyone else has a suggestion.

**Reply by dbuechel** [reply]
Thanks for the confirmation and for the hint, will do so. I have already searched the forum and not found exactly what we need, but I shall ask the question there again in the hope to maybe encounter an easy solution.

Otherwise we'll likely need to suppress it via a low-level hook (if that's possible for the mouse wheel) or then JavaScript we inject in every webpage.

**Reply by dbuechel** [reply]
To whom it may concern: Zooming with `CTRL + Mouse Wheel` should be suppressible via JavaScript, see https://www.magpcss.org/ceforum/viewtopic.php?f=6&t=19768 resp. https://stackoverflow.com/questions/56465207/how-can-i-prevent-full-page-zoom-on-mousewheel-event-in-javascript.
